Content.Shared/_DV/CosmicCult/SharedMonumentSystem.cs

```diff
@@ -1,4 +1,4 @@
-using Content.Shared._DV.Cosmiccult;
+using Content.Shared._DV.CosmicCult;
```

Comment:
The content of this file is all in `Content.Shared._DV.CosmicCult` already, so the using here is redundant